inspirehep/modules/workflows/workflows/article.py

```diff
@@ -236,13 +241,18 @@
 SEND_TO_LEGACY = [
     IF_ELSE(
         is_marked('is-update'),
+        IF_ELSE(
+            # feature flag to send updates to legacy or not
+            is_upload_to_legacy_on_update_enabled,
+            [
+                send_robotupload(mode='replace'),
+            ],
+            [
+                mark('skipped-robot-upload', True),
+            ]
+        ),
```

Comment:
I would suggest to use the same style of the already existing feature flags, like
```python
if not current_app.config.get('FEATURE_FLAG_ENABLE_FUZZY_MATCHER'):
        return False
````
which means inside the function itself and not in the workflow, WDYT?

Reply:
hmm IMO for ``SEND_TO_LEGACY`` and ``STORE_RECORD`` it's too much hidden logic, I'd rather state what they are doing in the workflow itself. WDYT?